src/dbnode/persist/fs/commitlog/commit_log.go

```diff
@@ -124,8 +124,32 @@ func (f *flushState) getLastFlushAt() time.Time {
 }
 
 type writerState struct {
-	writer     commitLogWriter
-	activeFile *persist.CommitLogFile
+	// See "Rotating Files" section of README.md for an explanation of how the
+	// primaryWriter and secondaryWriter fields are used during commitlog rotation.
+	primaryWriter   asyncResettableWriter
+	secondaryWriter asyncResettableWriter
```

Comment:
A lot of the code is really long below, maybe just call this `primary` and `secondary`?

Then calls will be like:
```go
l.writerState.primary.writer.Whatever(..)
```